src/app/shared/components/template/components/date-time-picker/date-time-picker.component.ts

```diff
@@ -0,0 +1,41 @@
+import { Component, computed, OnInit, signal, WritableSignal } from "@angular/core";
+import { TemplateBaseComponent } from "../base";
+import { getStringParamFromTemplateRow, parseBoolean } from "src/app/shared/utils";
+
+interface IDateTimePickerParams {
+  /** TEMPLATE PARAMETER: "disabled". If true, date time picker is disabled and greyed out */
+  disabled: boolean;
+  /** TEMPLATE PARAMETER: "type". Determines the type of picker date & time ('datetime'), date only ('date') or time only ('time') picker*/
+  type: string;
```

Comment:
nit(non-blocking): If any of the parameters are optional for authors then probably best to update the type definition accordingly, e.g.
```ts
  /** TEMPLATE PARAMETER: "disabled". If true, date time picker is disabled and greyed out */
  disabled?: boolean;
  /** TEMPLATE PARAMETER: "type". Determines the type of picker date & time ('datetime'), date only ('date') or time only ('time') picker*/
  type?: string;
```